lib/ts/station/registry.ex

```diff
@@ -40,6 +40,24 @@ defmodule Station.Registry do
     end
   end
 
+    # Returns the Query Collecotr pid associated with the query id
+    def lookup_query_id(qid) do
```

Comment:
unit tests are needed for this function.

Reply:
Test "Perform lookup for registered and unregistered query" has been added.